implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -183,11 +213,29 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
             contextInfo.setTokenHeader(tokenHeader);
         }
         SmallryeJwtUtils.setContextTokenCookie(contextInfo, tokenCookie);
+        if (defaultSubjectClaim != null && defaultSubjectClaim.isPresent()) {
+            contextInfo.setDefaultSubjectClaim(defaultSubjectClaim.get());
+        }
         if (defaultGroupsClaim != null && defaultGroupsClaim.isPresent()) {
             contextInfo.setDefaultGroupsClaim(defaultGroupsClaim.get());
         }
         SmallryeJwtUtils.setContextGroupsPath(contextInfo, groupsPath);
 
+        if (expGracePeriodSecs != null && expGracePeriodSecs.isPresent()) {
+            contextInfo.setExpGracePeriodSecs(expGracePeriodSecs.get());
+        }
+
+        if (whitelistAlgorithms != null && !whitelistAlgorithms.isEmpty()) {
+            contextInfo.getWhitelistAlgorithms().addAll(whitelistAlgorithms);
+        }
+
+        // JWKS based on jose4j
+        if (jwksLocation != null && jwksLocation.isPresent()) {
+            contextInfo.setJwksUri(jwksLocation.get());
+            contextInfo.setJwksRefreshInterval(jwksRefreshInterval);
+            contextInfo.setFollowMpJwt11Rules(false);
```

Comment:
Good catch, I'll need to check what is going on in Thorntail because that code is reachable from thorntail and is being tested

Reply:
Maybe you have another bean with a Specializes or Alternative to change the value? As is, the JwksVerificationKeyResolver is not reachable. I'm using Quarkus btw.

Reply:
Yes, something like that